compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -1013,19 +1015,19 @@ def cmd_start_endpoint(
             os.close(read_handle)
             exit_code += 1
 
-            log.debug("Redirecting stdout and stderr (%s)", os.devnull)
-            with os.fdopen(null_fd, "w") as null_f:
-                if os.dup2(null_f.fileno(), 1) != 1:
-                    raise OSError("Unable to close stdout")
+            log.debug("Convey credentials; redirect stdout, stderr (to '%s')", ep_log)
+            log_fd = os.open(ep_log, os.O_WRONLY | os.O_APPEND | os.O_SYNC, mode=0o200)
+            with os.fdopen(log_fd, "w") as log_f:
+                if os.dup2(log_f.fileno(), 1) != 1:
+                    raise OSError(f"Unable to direct stdout to {ep_log}")
```

Comment:
``to redirect`` ? Or did you shorten it to direct on purpose here and and below?

Reply:
No, not intentional.  Generally, "meh," as I don't expect this to be a problem, ~ever, and the intention is clear.  However, conformity is good, so "good catch."

Reply:
Fixed; ready for re-review.